src/components/view/ContactInfo/contactInfoEventsCard.js

```diff
@@ -0,0 +1,65 @@
+import React, { Component } from 'react';
+import ReactDOM from 'react-dom';
+
+import { Card, CardHeader, CardText } from 'material-ui/Card';
+import { List, ListItem } from 'material-ui/List';
+
+import { getImageForEnv } from 'static/images/index'
+
+class ContactInfoEventsCard extends Component {
+
+    formatUidToDate = uid => {
+        const splitUid = uid.split('');
+        return `${splitUid.slice(4,6).join('')}/${splitUid.slice(7,9).join('')}/${splitUid.slice(0,4).join('')}`
+    }
+
+    render() {
+        const { events } = this.props
+
+        const cardProps = {
+            initiallyExpanded: true
+        }
+
+        const cardHeaderProps = {
+            title: 'Events',
+            actAsExpander: true,
+            showExpandableButton: true,
+            titleColor: this.props.titleColor
+        }
+
+        const cardTextProps = {
+            expandable: true
+        }
+
+        const noEventsText = 'There are no events for this user'
+
+        return (
+            <Card {...cardProps}>
+                <CardHeader {...cardHeaderProps} />
+                <CardText {...cardTextProps}>
+                    {/* QUESTION: is it even worth using list and listItem if we need
```

Comment:
Generally, there are some cross-browser* benefits to using the MUI components instead of just native ones. Of course, there are always exceptions and I welcome the simplest solution that addresses the heart of the issue. Have you tried HTML lists and confirmed that it would work just as well?

#question

Reply:
It would totally work with standard lists, I used the MUI component because I imagine that in the future we're going to want the user to be able to interact with the list in some way so that they can go view the event, or take action on it .. and MUI looks to have nice built-in stuff for that, like swiping on a list item, adding buttons, logos, etc